components/mdc/Datatable/DataRow.svelte

```diff
@@ -1,3 +1,7 @@
-<tr class="mdc-data-table__row">
+<script>
+  export let clickable = false
+</script>
+
+<tr on:click on:keydown class="mdc-data-table__row {$$props.class}" class:pointer={clickable}>
```

Comment:
just wondering if the `on:click` needs to be blocked if `clickable` is false

Reply:
the event shouldn't do anything unless the parent is listening, eg: <Datatable.Data.row on:click={onClick}/> right? I suppose the consumer could forget to set clickable but that just controls the css. If we do need to block on:click is there a way other than manually dispatching an event? I didn't see any event modifiers that seemed to do this.

Reply:
Yeah, probably not worth the added complexity. Just thought I'd check with you.